portal-frontend/src/app/features/review-application/review-application.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Subject, takeUntil } from 'rxjs';
+import { ApplicationReviewService } from '../../services/application-review/application-review.service';
+import { ApplicationDto } from '../../services/application/application.dto';
+import { ApplicationService } from '../../services/application/application.service';
+
+@Component({
+  selector: 'app-review-application',
+  templateUrl: './review-application.component.html',
+  styleUrls: ['./review-application.component.scss'],
+})
+export class ReviewApplicationComponent implements OnInit, OnDestroy {
```

Comment:
Do we need to parameterise loading a specific step via routing?

Reply:
The requirement hasnt come up